src/reactable/thead.jsx

```diff
@@ -15,9 +15,19 @@ export class Thead extends React.Component {
             if (typeof th.props !== 'undefined') {
                 column.props = filterPropsFrom(th.props);
 
+                //set the label depending on the props
+                if(typeof th.props.label !== 'undefined'){
+                    column.label = th.props.label
+                }
+                else if(typeof th.props.children === 'string'){
+                    column.label = th.props.children
+                }
+                else {
+                    column.label = th.props.column
+                }
```

Comment:
the column has to be a string, see the code below. Perhaps better to leave this else block off it.

Could you also remove the first build commit and only have a final one?